### PR TITLE
490083 Add test for properties support

### PR DESCRIPTION
**Pull request by rawagner**
Signed-off-by: Rastislav Wagner <rawagner@redhat.com>

### CODE REVIEW

**Comment by fbricon** [comment]
You should also test properties (de)serialization in ResolverConfigurationIO.java

File: org.eclipse.m2e.tests/src/org/eclipse/m2e/tests/properties/MavenPropertiesTest.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/*******************************************************************************
+ * Copyright (c) 2010 Sonatype, Inc.
```
Comment (fbricon) [inline org.eclipse.m2e.tests/src/org/eclipse/m2e/tests/properties/MavenPropertiesTest.java]: 2016 Red Hat, Inc.

**Comment by rawagner** [comment]
Updated. Can you take a look ?

File: org.eclipse.m2e.tests/src/org/eclipse/m2e/tests/properties/MavenPropertiesTest.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    testProperties.put("key3", "");
+    testProperties.put("key4", "&#33|");
+    testProperties.put("key5", "<![CDATA[value|||]]>");
+    testProperties.put("key6", "<![CDATA[value>]]>");
+    testProperties.put("key7", "value%7C");
+
+    resolverConfiguration.setProperties(testProperties);
+
+    ResolverConfigurationIO.saveResolverConfiguration(project, resolverConfiguration);
+    ResolverConfiguration conf = ResolverConfigurationIO.readResolverConfiguration(project);
```
Comment (fbricon) [inline org.eclipse.m2e.tests/src/org/eclipse/m2e/tests/properties/MavenPropertiesTest.java]: can you check the properties actually read from the config file loaded from disk?

**Comment by fbricon** [comment]
Applied as https://github.com/tesla/m2e-core-tests/commit/bc4c6c2382819a9fe5bc4776b91ebe77b612949d after testing the loading of an existing resover config.

**Pull request closed without merging**